src/c++/library/common.h

```diff
@@ -400,7 +400,7 @@ class InferRequestedOutput {
   /// \return Error object indicating success or failure.
   static Error Create(
       InferRequestedOutput** infer_output, const std::string& name,
-      const size_t class_count = 0);
+      const std::string& datatype = "INT32", const size_t class_count = 0);
```

Comment:
I don't know that I love this default, but the alternative was to fix 100 cases in unrelated unit tests.

Reply:
crap, you're totally right. I'll need to add it at the end.